front-end/src/components/StampTotalTrend.vue

```diff
@@ -0,0 +1,123 @@
+// STEP 1: to include the dependencies
+<script>
+import Vue from 'vue';
+import VueFusionCharts from 'vue-fusioncharts';
+import FusionCharts from 'fusioncharts';
+import Column2D from 'fusioncharts/fusioncharts.charts';
+import FusionTheme from 'fusioncharts/themes/fusioncharts.theme.fusion';
+import Charts from "fusioncharts/fusioncharts.charts";
+import { FCComponent } from "vue-fusioncharts";
+import constants from "../constants";
+
+Charts(FusionCharts);
+Vue.use(VueFusionCharts, FusionCharts, Column2D, FusionTheme, FCComponent);
+const axios = require('axios').default;
+let api_server = 'http://localhost:3000';
+let scheme = '002'
+const dataStore={}
+
+
+export default {
+    name: 'app',
+  data() {
+    return {
+      type: "mscolumn2d",
+      renderAt: "chart-container",
+      width: constants.chart_width,
+      height: constants.chart_height,
+      dataFormat: "json",
+      dataSource: dataStore
+    }
+  },
+    mounted() {
+        this.getData()
+    },
+    methods: {
+        async getData() {
+
+            // Get the chart data
+            const formattedData = await axios.get(`${api_server}/stamp_total_Trend?scheme=${scheme}`)
+                .then(function (response) {
+                    // Handle success
+                    console.log("Here's the response")
+                    console.log(response.data)
+
+                    const dataStore = {
+                        chart: {
+                            caption: "Total Stamps Collected",
+                            paletteColors: constants.palette,
+                            subcaption: "PAV",
+                            xaxisname: "Years",
+                            yaxisname: "Total number of stamps",
+                            formatnumberscale: "1",
+                            plottooltext:
+                            "<b>$dataValue</b> stamps were collected for scheme <b>$seriesName</b> in $label",
+                            theme: "fusion",
+                            drawcrossline: "1"
+                        },
+                        categories: [
+                            {
+                            category: [
+                                {
+                                label: response.data.label1
+                                },
+                                {
+                                label: response.data.label2
+                                }
+                            ]
+                            }
+                        ],
+                        dataset: [
+                            {
```

Comment:
Is this hard-coded for two locations and schemes? It should work for any number of locations and schemes